src/NexusMods.Collections/Verbs.cs

```diff
@@ -107,4 +119,91 @@ private static async Task<int> GatherCollectionDefinitions([Injected] IRenderer
 
         return 0;
     }
+
+    [Verb("create-collection", "Creates a collection")]
+    private static async Task<int> CreateCollection(
```

Comment:
Please better define what the verb does. Does it create a writable collection? A read-only one? Does it create a collection on Nexus?